pkgs/development/r-modules/wrapper-radian.nix

```diff
@@ -12,8 +12,7 @@ runCommand (radian.name + "-wrapper") {
   preferLocalBuild = true;
   allowSubstitutes = false;
 
-  buildInputs = [ radian ] ++ recommendedPackages ++ packages
-    ++ lib.optional wrapR R;
+  buildInputs = [ R radian ] ++ recommendedPackages ++ packages;
```

Comment:
Is it still necessary to have R as part of buildInputs?

Reply:
It's not. Tested it out and it works without having R be there necessarily.

Reply:
Hmm, packages seem to be broken now and they were working before when R was part of buildInputs:

```
❯ nix shell --impure --expr 'with import ./. {}; radianWrapper.override {packages = with rPackages; [ggplot2];}'  -c radian
R version 4.2.3 (2023-03-15) -- "Shortstop Beagle"
Platform: x86_64-pc-linux-gnu (64-bit)

r$> library(ggplot2)
Error in library(ggplot2) : there is no package called ‘ggplot2’
```

I don't understand why though, `R_LIBS_SITE` is still being set.

Reply:
Strange. I tried this with both `ggplot2` and `dplyr` and it's working perfectly.